src/components/circuitDisplay/gateInfo.vue

```diff
@@ -164,6 +204,11 @@ export default {
               <chart-matrix :matrix="matrix" entry-type="complex" :display-title="false"></chart-matrix>
             </chart-def>
 
+            <div v-if="displayOp.type === 'ExpBox'">
+              <chart-def title="Phase" hover>
+                <mathjax-content :formula="'\`' + displayOp.box.phase + '\`'"></mathjax-content>
```

Comment:
An ExpBox has the form $e^{i t M}$ for some Hermetian matrix $M$ and real number $t$.

![Screenshot 2024-03-26 at 13 57 02](https://github.com/CQCL/pytket-circuit-renderer/assets/93673602/6cb75db0-18e0-407d-be67-363b86b45ceb)

Maybe we could use "Exponentiated Matrix" or similar to clarify that the matrix in the display is $M$ rather than $e^{i t M}$. This is a minor comment as I think this would be clear in most cases as the matrix $M$ is not unitary in general.

Reply:
Have updated the matrix titles for ExpBox and ProjectorAssertion box accordingly. I think all the other matrices displayed are either the actual unitary implemented, or displayed under a different name.